src/docs/product/alerts/alert-types.mdx

```diff
@@ -67,7 +67,7 @@ Crash rate alerts tell you when the crash free percentage for either [sessions o
 - Crash Free User Rate
 
 ### Performance
-
+For performance alerts, [metrics-based](/product/sentry-basics/search/searchable-properties/#metrics-properties) data is the default. If you make a selection in configuring your alert that's not metrics-compatible, then your alert will use event-based data and a message saying this will be displayed.
```

Comment:
```suggestion
For performance alerts, [metric-backed](/product/sentry-basics/search/searchable-properties/#metrics-properties) data is the default. If you make a selection in configuring your alert that's not metrics-compatible, then your alert will use indexed events, or event-based data, and a message saying this will be displayed.
```